client/components/date-range/shortcuts.tsx

```diff
@@ -40,6 +41,7 @@ const DateRangePickerShortcuts = ( {
 	const normalizedStartDate = startDate ? normalizeDate( startDate ) : null;
 	const normalizedEndDate = endDate ? normalizeDate( endDate ) : null;
 
+	// TODO: Receive this list from the parent component.
```

Comment:
I think the full shortcut list could possibly live in the component, and caller could just pass in the list of ids to get them configured.

Reply:
Both ways are good. We just need to ensure there is only one source of the shortcuts list.